Loenn/entities/arrowBubble.lua

```diff
@@ -0,0 +1,31 @@
+local arrowBubble = {}
+
+arrowBubble.name = "isaBag/arrowBubble"
+arrowBubble.depth = -100
+arrowBubble.placements = {
+    {
+        name = "Arrow Bubble (Left, IsaGrabBag)",
+        data = {
+            direction = left
+        }
+    },
+        name = "Arrow Bubble (Right, IsaGrabBag)",
+        data = {
+            direction = right
+        }
+    },
+        name = "Arrow Bubble (Up, IsaGrabBag)",
+        data = {
+            direction = up
+        }
+    },
+        name = "Arrow Bubble (Down, IsaGrabBag)",
+        data = {
+            direction = down
+        }
+    }
+}
+
+#TODO: add texture based on direction
```

Comment:
all the images are formatted as `booster{dir}00.png`, so you can use something like this:
`local texture = "isafriend/objects/booster/booster" .. dir .. "00"`